### PR TITLE
consistent chart.yaml for all kyma components

### PR DESCRIPTION
**Pull request by a-thaler**
**Description**

Changes proposed in this pull request:

- introduced a consistent content of chart.yaml for all kyma components
- added icon and home for kyma
- set chart version to 1.0.0 across all charts as putting the actual kyma version is not a trivial task for automation
- removed all other existing fields like maintainers (they can be looked up in CODEOWNERS) or appVersion (they are rarely maintained at all)

**Related issue(s)**

### CODE REVIEW

**Comment by k15r** [comment]
Would it make sense to add a proper description? Currently the generic "Kyma component '$componentname'" does not add any additional value.

**Comment by a-thaler** [comment]
@k15r as we cannot modify the name of the chart, it was important for me to indicate that it is an official kyma component. We might add a link to the docu for more info. I'm afraid that any description will not be maintained in long run.. That might change if we switch to real helm charts hosted in a helm repo..

**Comment by a-thaler** [comment]
@Szymongib I'm not sure. Sub-charts often contain 3party charts like for fluentbit, containing all the meta-data for the upstream chart. Sub-charts metadata will be not present on the cluster anyway, or?

File: resources/monitoring/Chart.yaml [hunk 1]
```diff
@@ -1,13 +1,7 @@
 apiVersion: v1
+apiVersion: v1
```
Comment (suleymanakbas91) [inline resources/monitoring/Chart.yaml:2]: `apiVersion` is duplicated.

Comment (a-thaler) [reply resources/monitoring/Chart.yaml]: fixed

File: resources/ory/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 apiVersion: v1
-appVersion: "1.0"
```
Comment (piotrmsc) [inline resources/ory/Chart.yaml:2]: With ory we in general want to put it 1:1 as much as possible with upstream chart

**Pull request merged**